client/src/pages/EpisodeShowPage/index.tsx

```diff
@@ -291,13 +290,9 @@ export class WorkspaceView extends React.Component<any, any> {
     ).allReadOnlyBlocks();
 
     let readOnlyExportedPointers = [];
-    try {
-      readOnlyExportedPointers = _.flatten(
-        allReadOnlyBlocks.map(b => findPointers(b.value))
-      );
-    } catch (err) {
-      LogRocket.captureException(err);
-    }
+    readOnlyExportedPointers = _.flatten(
```

Comment:
Does FullStory have a way to log errors? This line has unpredictably thrown errors. We want these errors caught because (1) they don't cause any downstream problems and (2) we don't want the app to crash for users during experiments. We do want to log any error here though so we can figure out what's going on.

Reply:
hmm, it looks like you can send strings, so not sure if that's going to be enough: https://help.fullstory.com/develop-js/251886